reference/4.0/PSDiagnostics/Get-LogProperties.md

```diff
@@ -1,41 +1,55 @@
 ---
-ms.date:  06/09/2017
-schema:  2.0.0
-locale:  en-us
-keywords:  powershell,cmdlet
-external help file:  PSDiagnostics-help.xml
+external help file: PSDiagnostics-help.xml
+Module Name: PSDiagnostics
+online version:
+schema: 2.0.0
+ms.date:  11/27/2018
+Help Version: 6.1.0.2
```

Comment:
Is this the right help version for a 4.0 doc?  Another 4.0 doc here has 4.0.0.1 as the value.  This is true of several docs in the PR.

Reply:
fixed